stories/checkboxes.stories.js

```diff
@@ -0,0 +1,59 @@
+/**
```

Comment:
@asvinb could you, please, add an appropriate file header?

Reply:
Updated @eugene-manuilov

Reply:
Awesome! Thanks.